### DISCUSSION TITLE
[question] Configuration file

### DISCUSSION THREAD

**Discussion by jagjordi** [librespot-org/librespot#1287] [General]
Is it possible to use a configuration file to pass the options to librespot?
In the old librespot-java it was possible. Also I saw a reference in a 2018 issue to config.toml
Was this option removed?

**Comment by roderickvd** [comment]
No, not possible. Some downstream packages like Raspotify might allow that, I don't know.

**Comment by kingosticks** [comment]
If I remember correctly, all the options can be set via env variables. You can set them all in a file and source that before starting librespot. I think that's what's raspotify does.

**Comment by jagjordi** [comment]
Is this documented somewhere? I could not find it myself

**Reply by kingosticks** [reply]
Under the table at https://github.com/librespot-org/librespot/wiki/Options
